09-AzureDevOps(VSTS)-Security/Readme.md

```diff
@@ -211,6 +212,172 @@ Step 1,2 & 3 needs to be repeated to add “__Project Component Security Scan Su
+and, after the process is performed once, AzSK.AzureDevOps remembers it and generates an effective control result for subsequent control scans _until_ there 
+is a state change.
+
+The attestation feature is implemented via a new switch called *ControlsToAttest* which can be specified in any of the standard security scan cmdlets
+of AzSK.AzureDevOps. When this switch is specified, the toolkit first performs a scan of the target resource(s) like it is business as usual and, once
+the scan is complete, it enters a special interactive 'attest mode' where it walks through each resource and relevant attestable controls
+and captures inputs from the user and records them (along with details about the person who attested, the time, etc.). 
+After this, for all future scans on the resource(s), AzSK.AzureDevOps will show the effective control evaluation results. Various options are provided to support
+different attestation scenarios (e.g., expiry of attestations, edit/change/delete previous attestations, attest only a subset of controls, etc.). 
+These are described below. Also, for 'stateful' controls (e.g., "are these the right service accounts with access to project collection?"), the attestation
+state is auto-reset if there is any change in 'state' (e.g., someone adds a new service account to the list).
+
+Lastly, due to the governance implications, the ability to attest controls is available to a subset of users. This is described in
+the permissions required section below.  
+
+
+[Back to top...](Readme.md#contents)
+### Starting attestation
+      
+The AzSK.AzureDevOps scan cmdlets now support a new switch called *ControlsToAttest*. When this switch is specified, 
+AzSK.AzureDevOps enters the attestation workflow immediately after a scan is completed. This ensures that attestation is done on the basis of the most current
+control status.
+
+All controls that have a technical evaluation status of anything other than 'Passed' (i.e., 'Verify' or 'Failed' or 'Manual' or 'Error') are considered 
+valid targets for attestation.
+
+> **Note**: Some controls are very crucial from security stand point and hence AzSK.AzureDevOps does not support attesting them.
+
+To manage attestation flow effectively, 4 options are provided for the *ControlsToAttest* switch to specify which subset of controls to target for attestation. These are described below:
+
+|Attestation Option|Description|
+|------------------|-----------|
+|NotAttested|Attest only those controls which have not been attested yet.|
+|AlreadyAttested|Attest those controls which have past attestations. To re-attest or clear attestation.|
+|All|Attest all controls which can be attested (including those that have past attestations).|
+|None|N/A.|
+
+For example, to attest organization controls, run the command below:
+```PowerShell  
+$subscriptionId = <Your SubscriptionId>
+Get-AzSKAzureDevOpsSecurityStatus -OrganizationName $orgName -ControlsToAttest NotAttested -ResourceTypeName Organization  
```

Comment:
Add cmd for project also
Get-AzSKAzureDevOpsSecurityStatus -OrganizationName $orgName -ProjectNames $prgNames -ControlsToAttest NotAttested -ResourceTypeName Project

Reply:
done